cms/templates/partials/catalog-card.html

```diff
@@ -0,0 +1,89 @@
+{% load static humanize wagtailcore_tags wagtailimages_tags %}
+<div class="catalog-card" role="region" aria-label="{{ courseware_page.title }}">
+    <div class="top">
+        <div class="cover-image">
+            <div class="cover-image-frame">
+                {% if courseware_page.thumbnail_image %}
+                    {% image courseware_page.thumbnail_image fill-335x203 %}
+                {% else %}
+                    <img src="{% static default_image_path %}" alt="Preview image">
+                {% endif %}
+            </div>
+        </div>
+        <div class="top-level-details">
+            <h3 class="title">
+                    <a href="{% pageurl courseware_page %}">
+                        {{ courseware_page.title }}
+                    </a>
+            </h3>
+            <ul>
+                {% if courseware_page.product.current_price %}
+                    <li>
+                        <strong>Cost:</strong>
+                        ${{ courseware_page.product.current_price|floatformat:"0"|intcomma }}
+                    </li>
+                {% endif %}
+                {% if object_type == "program" %}
+                    <li>
+                        <strong>Number of Courses:</strong>
+                        {{ courseware_page.product.courses.count }}
+                    </li>
+                {% endif %}
+                {% if courseware_page.duration %}
+                    <li>
+                        <strong>Duration:</strong>
+                        {{ courseware_page.duration }}
+                    </li>
+                {% endif %}
+                {% if courseware_page.product.next_run_date %}
+                    <li>
+                        <strong>Next Start Date:</strong>
+                        {{ courseware_page.product.next_run_date|date:"F j, Y" }}
+                    </li>
+                {% endif %}
+            </ul>
+        </div>
+        <div class="offering-type">
+            <label>
+                {% if object_type == "course" %}
+                    Single
+                {% else %}
+                    Multi
+                {% endif %}
+                Course Online
+            </label>
+        </div>
+    </div>
+    <div class="bottom">
+      <input type="checkbox" class="toggle" id="collapsible_{{ tab }}_{{ object_type }}_{{ courseware_page.pk }}" data-target="#collapse-{{ tab }}-{{ object_type }}-{{ courseware_page.pk }}" data-toggle="collapse" />
```

Comment:
Seems like we should use the ['with' templatetag](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#with) to define these complex id's rather than redefining them. Also, wouldn't it be sufficient just to define them as something like `collapsible_{{ courseware_page.pk }}`? Shouldn't the pk value be unique for each object?

Reply:
Updating to `collapsible_{{ tab}}_{{ coursewate_page.pk }}` as the same page can appear in another tab and we don't want to be toggling in both tabs at once.